WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -0,0 +1,294 @@
+#import "InsightsTableViewController.h"
+#import "WPFontManager+Stats.h"
+#import "WPStyleGuide+Stats.h"
+#import "StatsTableSectionHeaderView.h"
+#import "StatsGaugeView.h"
+
+@interface InlineTextAttachment : NSTextAttachment
+
+@property CGFloat fontDescender;
```

Comment:
Note here: This property will be handled as *atomic, assign, readwrite*. Do we need this one to be atomic?